### PR TITLE
Feedback changes

### PR DESCRIPTION
**Pull request by nitinprojects**
- Updated import statements with alias
- Removed empty label passed to Dropdown component
- Imported the third-party entities under an alias
- Used backslash to escape quotation marks
- Wrapped title with double quotes
- Changed prop name from data to notes
- Updated component names
- Fixed the error messages
- Added util to build select options
- Used buildSelectOptions util to build select options
- Added type prop to cancel buttons in the form
- dayjs related changes

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
+import { buildSelectOptions } from "utils";
 import * as yup from "yup";
 
 export const NOTES_LIST_DATA = [
   {
     id: 1,
     title: "How to claim the warranty?",
-    description: `Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`,
+    description:
+      '"Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn\'t getting',
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:9]: Not sure why you've enclosed this whole string within single quotes. This should be enclosed within double quotes. Go through the string and see if the double quotes already within the string add any value. I feel it doesn't add any value.

File: app/javascript/src/utils/index.js [hunk 2]
```diff
@@ -1,3 +1,10 @@
+import camelCase from "lodash/camelCase";
```
Comment (yedhink) [inline app/javascript/src/utils/index.js:1]: Don't use `lodash`. In BB we don't use lodash. Either use [Ramda](https://academy.bigbinary.com/learn-react/pages/learn-ramda-js) or create this util function using vanilla JS.

**Review by yedhink — COMMENTED** [review]
@nitinprojects Few comments.

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 1 above)
Comment (nitinprojects) [reply app/javascript/src/components/Dashboard/Notes/constants.js:9]: I followed the figma design.  I will remove the double quotes within the string.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/constants.js:9]: Thanks for pointing that out. Will update the docs.

**Pull request merged**